docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -8,7 +8,8 @@
 |spring.cloud.config.override-system-properties | true | Flag to indicate that the external properties should override system properties. Default true.
 |spring.cloud.discovery.client.composite-indicator.enabled | true | Enables discovery client composite health indicator.
 |spring.cloud.discovery.client.health-indicator.enabled | true | 
-|spring.cloud.discovery.client.health-indicator.include-description | false | 
+|spring.cloud.discovery.client.health-indicator.include-description | false |
+|spring.cloud.discovery.client.health-indicator.use-services-query | true | Whether the indicator should query for all registered services or use probe method.
```

Comment:
This file gets auto-generated so we should not modify it.  Add the description in the javadocs of the properties file.  Also it might be useful to add some documentation around how this property should be used in conjunction with `DiscoveryClient.probe`

Reply:
Oh thats awesome! **TIL**. Will do.

Reply:
@ryanjbaxter do you need anything else changed on this one?